contrib/android/Dockerfile

```diff
@@ -154,23 +165,29 @@ RUN chown --recursive ${USER} ${WORK_DIR} ${ANDROID_SDK_HOME}
 RUN chown ${USER} /opt
 USER ${USER}
 
+# venv, VIRTUAL_ENV is used by buildozer to indicate a venv environemnt
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3 -m venv ${VIRTUAL_ENV}
+ENV PATH="${VIRTUAL_ENV}/bin:${PATH}"
 
 COPY contrib/deterministic-build/requirements-build-base.txt /opt/deterministic-build/
 COPY contrib/deterministic-build/requirements-build-android.txt /opt/deterministic-build/
-RUN python3 -m pip install --no-build-isolation --no-dependencies --user \
+RUN /opt/venv/bin/python3 -m pip install --no-build-isolation --no-dependencies \
     -r /opt/deterministic-build/requirements-build-base.txt
-RUN python3 -m pip install --no-build-isolation --no-dependencies --no-binary :all: --user \
+RUN /opt/venv/bin/python3 -m pip install --no-build-isolation --no-dependencies --no-binary :all: \
     -r /opt/deterministic-build/requirements-build-android.txt
 
 # install buildozer
 RUN cd /opt \
     && git clone https://github.com/kivy/buildozer \
     && cd buildozer \
     && git remote add sombernight https://github.com/SomberNight/buildozer \
+    && git remote add accumulator https://github.com/accumulator/buildozer \
     && git fetch --all \
-    # commit: from branch sombernight/electrum_20210421 (note: careful with force-pushing! see #8162)
-    && git checkout "6f03256e8312f8d1e5a6da3a2a1bf06e2738325e^{commit}" \
-    && python3 -m pip install --no-build-isolation --no-dependencies --user -e .
+    # commit: from branch master (note: careful with force-pushing! see #8162) \
+    # no, this is master
+    && git checkout "698b48136c55b2495128207c5e98dccc128bb6c2^{commit}" \
```

Comment:
looks like you lost commit https://github.com/SomberNight/buildozer/commit/8791a52a02d1c6279504f402098bf6536824ea1c in a rebase/etc, and then reimplemented it yourself in https://github.com/SomberNight/buildozer/commit/698b48136c55b2495128207c5e98dccc128bb6c2 ? >.<

Mine had some references in the commit message, which might be interesting to track.
Also I have an open PR at https://github.com/kivy/buildozer/pull/1511 though I guess there hasn't been much interest in it (maybe we are doing something weird, not sure how this does not affect more people?).

I don't care which commit we use, but I think we should add a reference in the commit msg to https://github.com/kivy/buildozer/pull/1511.

Reply:
Argh, sorry, I must have force-pushed a older branch when I switched to my laptop >.<
I will repair tomorrow